### PR TITLE
WIP: Initial Draft of Remove ServiceMatrix from Getting Started with the Particular Service Platform

### PR DESCRIPTION
**Pull request by petersgiles**
Related to https://github.com/Particular/PlatformDevelopment/issues/454
Related to https://github.com/Particular/docs.particular.net/issues/856

I looking at this I think we are going to have to make a sample based on
what service matrix did.

@indualagarsamy There's a fair bit of work here and I wanted to check it in where I was before I finished today.

I'm working on a sample project that will setup a Nancy web site I don't think I'll get into how that's done in the documentation but there will be doco on a PostHandler method that would demonstrate how to load a message onto the bus

### CODE REVIEW

File: platform/getting-started-with-particular-service-platform.md [hunk 1]
```diff
@@ -321,4 +213,7 @@ What's so exciting about that?  After all inter-process communication has been d
 
 One answer is ***fault tolerance***.  Next we'll explore the fault tolerance and durability features NServiceBus offers.
 
-Continue on and let's review the fault tolerance in our [next article](/servicematrix/getting-started-with-nservicebus-using-servicematrix-2.0-fault-tolerance.md "Getting Started with Fault Tolerance").  
+
+
+#mORE sTUFF TO FIX
```
Comment (SimonCropp) [inline platform/getting-started-with-particular-service-platform.md]: > mORE sTUFF TO FIX 

?

**Comment by petersgiles** [comment]
@SimonCropp & @indualagarsamy This document is beginning to feel like this sample http://docs.particular.net/samples/step-by-step/

**Comment by petersgiles** [comment]
@SimonCropp @indualagarsamy any comments? I guess when I made the previous comment I was asking do we need this document all all if the step by step sample does the same thing? what value does this add now? should we take this in another direction?

**Comment by petersgiles** [comment]
After reviewing again I'm certain there is very little value in having this page now at all.

**Comment by SimonCropp** [comment]
ok closing this. can be revisited ltr

**Pull request closed without merging**